manifests/prow-test-pods/roles/deck.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: deck
+  namespace: prow-test-pods
```

Comment:
do we need to specify the ns? is not argo managing it? it applies to all yamls

Reply:
Is managed by argo, but this being isolated from the `3scale/platform` repo made more sense to me having the namespace explicit.

Reply:
ok, fair enough!